### PR TITLE
Made ut_runner to fail on invalid path

### PR DESCRIPTION
**Pull request by Pazus**
Made ut_runner to fail if non-existent schema is provided or provided package is not a suite package.

Refers #134

### CODE REVIEW

File: source/core/ut_suite_manager.pkb [hunk 1]
```diff
@@ -411,6 +411,8 @@ create or replace package body ut_suite_manager is
     for i in 1 .. l_paths.count loop
       l_path   := l_paths(i);
       l_schema := regexp_substr(l_path, '^(\w+)(\.|:|$)', 1, 1, null, 1);
+      
+      l_schema := sys.dbms_assert.schema_name(upper(l_schema));
```
Comment (jgebal) [inline source/core/ut_suite_manager.pkb]: Nice, I didnt use dbms_assert before. Good to know about it's existence.

File: source/core/ut_suite_manager.pkb [hunk 2]
```diff
@@ -434,6 +436,11 @@ create or replace package body ut_suite_manager is
             l_procedure_name := regexp_substr(l_path, '^\w+\.(\w+)(\.(\w+))?$', subexpression => 3);
 
             l_temp_suite := config_package(l_schema, l_package_name);
+
```
Comment (jgebal) [inline source/core/ut_suite_manager.pkb]: Maybe use dbms_assert.sql_object_name first, and if it fails, we know the object name is not valid.
Then do the if statement, and raise error stating, that the package is not a properly annotated unit test suite package.

**Comment by Pazus** [comment]
I've tested it manually but its not clear to write automated tests for that as we still have problem with `purge` method dynamic call which doesn't exists

**Comment by Pazus** [comment]
It failes on wrong schema, it failes on non-existent package and if provided package is not a suite package

**Comment by Pazus** [comment]
#161

**Comment by jgebal** [comment]
Please remove the purge from `run`. I will add it back properly implemented with changes to outputs.

**Comment by jgebal** [comment]
Never mind. I'll be making changes to this code anyway.

**Pull request merged**